### ISSUE TITLE
Saving Issue

### ISSUE THREAD

**Issue by Georggge** [Lunatrius/Schematica#302]
hey, recently ive been trying to save schematics but im having an issue, its not saving my entire selection, it keeps cutting it off for some reason?
Heres an image so you know what I mean
http://prntscr.com/i2fpti

also heres a picture of the mods im running incase thats of any importance
http://prntscr.com/i2fqaj

**Comment by Pokechu22** [comment]
That selection looks pretty large.  Are the chunks loaded when you try to save it?

**Comment by Georggge** [comment]
yeah, I first noticed it when trying to save a schematic of a pixelmon server spawn, even a little section of it some of it always seemed to cut off that picture was just to show an example

**Comment by Lunatrius** [comment]
That's only 3-4 chunks rendered, which shouldn't be anywhere close to a chunk limit, unless you have the render distance set to a very low value.

**Comment by Georggge** [comment]
nah I have it on max, never had the issue before and ive done very large schematic copies so I dont know what could be causing it, loads about 5-6 chunks

**Comment by Lunatrius** [comment]
What happens if you save the exact same region multiple times, restarting/relogging in between attempts?

**Comment by Georggge** [comment]
just saves the same area, no difference but only does about half of the selected section

**Comment by Lunatrius** [comment]
Can you test this with only Schematica installed as well? And the exact coordinates for the area that you're saving (blue/red point) would be great.
